src/scripts/main.js

```diff
@@ -1,5 +1,35 @@
 'use strict';
 
-document.addEventListener('click', e => {
-  // write code here
+const wall = document.querySelector('.wall');
+const spider = document.querySelector('.spider');
+
+wall.addEventListener('click', event => {
+  if (!event.target.closest('.wall') || event.target === spider) {
```

Comment:
`!event.target.closest('.wall')` - no need to check this, you've already put an event listener on the wall, the event will be triggered only inside the wall area